chalice/app.py

```diff
@@ -72,6 +77,16 @@ def version():
 
     @app.route("/internal/health")
     def health():
+        # Level 2 healthcheck: Test connection can be made to redshift cluster but do not run any queries
+        redshift_handler.transaction([])
+        # Level 2 healthcheck checks that ecs query runner is active with expected number of tasks
+        service_name = f"matrix-service-query-runner-{os.environ['DEPLOYMENT_STAGE']}"
+        service = ecs_client.describe_services(cluster=service_name, services=[service_name])["services"][0]
+        status = service["status"]
+        desired_task_count = service["desiredCount"]
+        running_task_count = service["runningCount"]
+        assert status == 'ACTIVE'
+        assert desired_task_count == running_task_count
```

Comment:
On fail, will this simply return a 500 to the client?

Reply:
Yea